code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -713,6 +713,7 @@
 
 /datum/outfit/syndicatespace/syndicaptain
 	name = "Syndicate Ship Captain"
+	uniform = /obj/item/clothing/under/syndicate/tacticool
```

Comment:
why the fake tacticool one and not the combat/regular one the other guys get

Reply:
I just figured that it should be different to make them stand out. I'll make it the combat one though on second thought.